qa/L0_perf_tfs/test.sh

```diff
@@ -58,7 +60,7 @@ mkdir -p model_store/${MODEL_NAME}
 cp -r ${REPODIR}/perf_model_store/${MODEL_NAME}/1/model.savedmodel model_store/${MODEL_NAME}/1
 
 # Run server
-tensorflow_model_server --port=8500 --model_name=${MODEL_NAME} --model_base_path=$PWD/model_store/${MODEL_NAME} > server.log 2>&1 &
+tensorflow_model_server --port=8500 --model_name=${MODEL_NAME} --model_base_path=$PWD/model_store/${MODEL_NAME} --backend-config=tensorflow,version=${TF_VERSION} > server.log 2>&1 &
```

Comment:
Isn't this TF Serving? Does TF Serving accept this `--backend-config=tensorflow,version=${TF_VERSION}` argument? Or is it Triton-specific?

Reply:
Yes I'm checking the flag as well and you are right, it won't throw an error for non existing arg. I will revert the changes from here. Thank you Ryan!